DOMManipulation.js

```diff
@@ -0,0 +1,8 @@
+//Using JavaScript, create a button, add it to the html
+let div = document.querySelector('.selfAssesment');
+let button = document.createElement('BUTTON');
+let text = document.createTextNode("Button");
+button.appendChild(text);
+div.appendChild(button);
```

Comment:
This was a great start! Next step would have been to add an event listener to this button. In that function you would have had to create another element, a `<h1>`, and to disable the button + change the body's background color.